modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/ColorField.testcase

```diff
@@ -32,24 +32,6 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-98070. As a Developer, I want to Implement the color field type"
```

Comment:
Hi @thaisfrota , I'll choose to keep this test in order to have the specific result for this test scenario.